project/Makefile.twig

```diff
@@ -2,7 +2,7 @@
 #
 # It's auto-generated by sonata-project/dev-kit package.
 
-.PHONY: test docs
+.PHONY: test docs{% if assets_path %} assets{% endif %}
```

Comment:
It would be great to avoid the twig template of the Makefile.

Could we play another way? Maybe generic setup task and/or env vars?

Reply:
Maybe switch to [composer scripts](https://github.com/sonata-project/dev-kit/issues/263)?

Reply:
This is front building.

Why not make a yarn custom script and call it on travis bash files?

Reply:
Will this change anything? We still need to call `yarn run assets`.

The `Makefile` is our main file for all automated jobs (e.g. phpunit, docs, etc.), so we should add this yarn stuff too.

Reply:
> Will this change anything?

Yes: https://github.com/sonata-project/dev-kit/pull/213/files#r140481910